UnityProject/Assets/Scripts/Systems/Lobby/CharacterSheet.cs

```diff
@@ -110,11 +110,16 @@ private void ValidateAiName()
 
 	public void ValidateSpeciesCanBePlayerChosen()
```

Comment:
this needs to stay the same because, otherwise then slimes break

Reply:
How does it break aliens and slimes? as far as I'm aware, this only affects the character creator, and isn't called elsewhere.